docs/runtimes/configuring-job-resources.md

```diff
@@ -252,38 +254,29 @@ fn.with_node_selection(node_selector={"app.iguazio.com/lifecycle":"non-preemptib
-scheduling: a lower priority pod can be evicted to allow scheduling of a higher priority pod. Pod priority is relevant for all pods created 
-by the service. For MLRun, it applies to the jobs created by MLRun. For Nuclio it applies to the pods of the Nuclio-created functions.
-
-Eviction uses these values in conjunction with pod priority to determine what to evict [Pod Priority and Preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption).
-
-Pod priority is specified through Priority classes, which map to a priority value. The priority values are: High, Medium, Low. The default is Medium. Pod priority is supported for:
-- MLRun jobs: the default priority class for the jobs that MLRun creates.
-- Nuclio functions: the default priority class for the user-created functions.
-- Jupyter
-- Presto (The pods priority also affects any additional services that are directly affected by Presto, for example like hive and mariadb, 
-which are created if Enable hive is checked in the Presto service.)
-- Grafana
-- Shell
+See:
+- py:meth}`~#mlrun.runtimes.RemoteRuntime.with_priority_class`
+- {py:meth}`~#mlrun.runtimes.RemoteRuntime.with_node_selection`
 
 ### UI configuration
 
 ```{admonition} Note
 Relevant when MLRun is executed in the [Iguazio platform](https://www.iguazio.com/docs/latest-release/).
 ```
-Configure the default priority for a service, which is applied to the service itself or to all subsequently created user-jobs in the 
-service's **Common Parameters** tab, **User jobs defaults** section, **Priority class** drop-down list.
 
-Modify the priority for an ML function by pressing **ML functions**, then **<img src="../_static/images/kebab-menu.png" width="25"/>** 
-of the function, **Edit** | **Resources** | **Pods Priority** drop-down list.
+You can configure Spot node support when creating a job, rerunning an existing job, and creating an ML function. 
+The **Run on Spot nodes** drop-down list is in the **Resources** section of jobs. 
+Configure the Spot node support for individual Nuclio functions when creating a function in the **Configuration** tab, under **Resources**. 
+
+## Pod priority for user jobs
+
+[Priority classes](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/) are a mechanism in Kubernetes to 
+control the order in which pods are scheduled and evicted &mdash; to make room for other, higher priority pods. Priorities also affect the pods’ 
+evictions in case the node’s memory is pressured (called Node-pressure Eviction).
 
+Pod priority is relevant for all of the jobs created by MLRun. For Nuclio it applies to the pods of the Nuclio-created functions.
+
+Pod priority is specified through Priority classes, which map to a priority value. The priority values are: High, Medium, Low. The default is Medium. 
```

Comment:
> The priority values are: High, Medium, Low. The default is Medium. 

That's not always true. they can use `fn.list_valid_priority_class_names()` and `fn.get_default_priority_class_name()` to get these values.